ui/src/containers/organisations.list/serviceusers/details.tsx

```diff
@@ -0,0 +1,120 @@
+import { Flex, Grid, Text } from "@raystack/apsara";
+import { V1Beta1Organization, V1Beta1ServiceUser } from "@raystack/frontier";
+import { useFrontier } from "@raystack/frontier/react";
+import dayjs from "dayjs";
+import { useEffect, useState } from "react";
+import { Link, Outlet, useParams } from "react-router-dom";
+import PageHeader from "~/components/page-header";
+import { DEFAULT_DATE_FORMAT } from "~/utils/constants";
+
+type DetailsProps = {
+  key: string;
+  value: any;
+};
+
+export default function ServiceUserDetails() {
+  let { organisationId, serviceUserId } = useParams();
+  const [organisation, setOrganisation] = useState<V1Beta1Organization>();
+  const [serviceUser, setServiceUser] = useState<V1Beta1ServiceUser>();
+
+  const { client } = useFrontier();
+
+  const pageHeader = {
+    title: "Organizations",
+    breadcrumb: [
+      {
+        href: `/organisations`,
+        name: `Organizations list`,
+      },
+      {
+        href: `/organisations/${organisation?.id}`,
+        name: `${organisation?.title}`,
+      },
+      {
+        href: `/organisations/${organisation?.id}/serviceusers`,
+        name: "Service Users",
+      },
+      {
+        href: `/organisations/${organisation?.id}/serviceusers/${serviceUser?.id}`,
+        name: serviceUser?.title || "",
+      },
+    ],
+  };
+
+  useEffect(() => {
+    async function getOrganization() {
+      const {
+        // @ts-ignore
+        data: { organization },
+      } = await client?.frontierServiceGetOrganization(organisationId ?? "");
+      setOrganisation(organization);
+    }
+
+    async function getServiceUser(userId: string) {
+      const resp = await client?.frontierServiceGetServiceUser(userId);
+      const user = resp?.data?.serviceuser;
+      setServiceUser(user);
+    }
+    if (organisationId) {
```

Comment:
Didn't quite get why we're fetching the `organizationId` here. Is it to show the org name in breadcrumbs?

Reply:
yes, but this can also check if org exist or not.

Reply:
Do we need to check that though? 
In the absence of an org, the page navigation (using org ->service users->user details would not work. 
If someone directly opens this page, won't the service user fetch anyway fail because the user would be detached from org?